docs/api.md

```diff
@@ -0,0 +1,239 @@
+# API Documentation
+
+## Things to consider
+* An upvotes table to keep track of user upvotes
+    * Useful to prevent a single user upvoting multiple times
+* Should universities have data
+    * Flavour text ect.
+    * Some kind of reuptation or something?
+
+## Identifying Data Types
+* Users
+* Universities
+* Courses
+* Questions
+* Answers (Replies to Questions)
+* Reviews
+* Replies (Replies to Reviews)
+
+## Sample Schema's (json)
+
+### User
+```
+"user": {
+    "uid": <unique int>
```

Comment:
all ids inside objects are implicity the type of id, no need to make them specifc. Will just make frontend logic more complex.

Reply:
eg. User
```json
{
   "id" : "xxxx",
   "username" : "xxxx",
   "meta" : { }
}
```